### PR TITLE
Release Day II

### PR DESCRIPTION
**Pull request by reactivepixel**
All devs please verify that your code and functionality that has been worked on is present in this release as you intend.

### CODE REVIEW

File: bot/controllers/verify.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
+const models = require('../../db/models');
+const uuidv4 = require('uuid/v4');
+const util = require('apex-util');
+const nodemailer = require('nodemailer');
+require('dotenv').config();
```
Comment (reactivepixel) [inline bot/controllers/verify.js]: envs should be added to max.config.js. Loading in the env file like this is redundant.

File: bot/controllers/verify.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      allowInDM: false,
+      resType: 'reply',
+      action: (message, ctrl, msg) => {
+        const validDomains = ['student.fullsail.edu', 'fullsail.edu'];
+        const email = msg.parsed[1].toLowerCase();
+        const emailDomain = email.split('@').pop();
+        // Method to generate random numeric verification code
+        // Modified to fit style guide from this SO answer:
+        // https://stackoverflow.com/a/39774334
+        const generateCode = (n) => {
```
Comment (reactivepixel) [inline bot/controllers/verify.js]: A utility function like this should be reusable and defined outside of the scope of this ctrl

File: bot/controllers/verify.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+            from: 'discord.maxbot@gmail.com',
+            to: email,
+            subject: 'Armada Verification Code',
+            html: `<p>Verification Code: ${code}</p>`,
+          };
+          // Call sendMail on sendVerifyCode
+          // Pass mailOptions & callback function
+          sendVerifyCode.sendMail(mailOptions, (err, info) => {
+            if (err) {
+              message.reply('Oops, looks like the email can not be sent. Its not you, its me.. Please contact a moderator and let them know I have failed.');
```
Comment (reactivepixel) [inline bot/controllers/verify.js]: Messages should be replying with variables defined at the top of the action callback for more comprehensible future editing of the messages.

File: bot/controllers/verify.js (hunk 1 above)
Comment (Rhuansantos) [reply bot/controllers/verify.js]: Fixed!

File: bot/controllers/verify.js (hunk 2 above)
Comment (Rhuansantos) [reply bot/controllers/verify.js]: Fixed!

File: bot/controllers/verify.js (hunk 3 above)
Comment (ghost) [reply bot/controllers/verify.js]: Thank you, message has been stored in variable to resolve this issue.

File: bot/controllers/verify.js (hunk 1 above)
Comment (ghost) [reply bot/controllers/verify.js]: Thank you, the env file logic that was implemented has been changed.

File: bot/controllers/verify.js (hunk 2 above)
Comment (ghost) [reply bot/controllers/verify.js]: Thank you, the utility function is now reusable and resides outside of the ctrl's scope.

**Review by ghost — APPROVED** [review]
3 issues were mentioned and resolved.

LGTM :+1:

**Comment by Rhuansantos** [comment]
We are ready to go now!
I have tested all the process and LGTM 👍 🥇 

- [x] !Verify user
- [x] Node Mailer - with the code generator
- [x] Storing the user verification in the database

👍 🥇 💯

**Comment by ghost** [comment]
# Release 2
## !verify front-end functionality of Max-Bot
### Expectations Accomplished 
#### User !verify
* Obtain email address
* Send verification code to user email
* User sends verification code to Max
* Max acknowledges matching code a prepares to send "over the bridge"
#### Email
* Nodemailer setup and functioning
#### Database
* MariaDB structured
* Tables made

## Special thanks
Thank you to all the team members that worked tirelessly in this release. Relax and get some rest for tomorrow it all starts again :joy:

**Comment by ghost** [comment]
# LGTM :+1:

**Pull request merged**